src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -2140,15 +2140,6 @@ public boolean handleMuteRequest(Jid fromJid, Jid toBeMutedJid, boolean doMute,
                     && participantChannels != null
                     && bridgeSession.colibriConference.muteParticipant(participantChannels, doMute, mediaType);
 
-        if (succeeded)
```

Comment:
(maybe explained later): how is it we can get rid of this muted state? were we not using it anywhere?

Reply:
Nope. It was only written, and I don't think it was consistent either.